arches_references/models.py

```diff
@@ -274,7 +282,7 @@ def serialize(self):
             "valuetype_id": self.valuetype_id,
             "language_id": self.language_id,
             "value": self.value,
-            "list_item_id": self.list_item_id,
+            "list_item_id": str(self.list_item_id),
```

Comment:
Can you make the identical change on ListItemImage? It's an unmanaged model that provides a Django FileField on the value column but is backed by the same database table as ListItemValue.